pandas/tests/arrays/string_/test_string.py

```diff
@@ -345,3 +345,22 @@ def test_astype_from_float_dtype(dtype):
     result = s.astype("string")
     expected = pd.Series(["0.1"], dtype="string")
     tm.assert_series_equal(result, expected)
+
+
+def test_contains():
+    # GH-37867
+    arr = pd.arrays.StringArray(np.array(["a", "b"], dtype=object))
```

Comment:
```suggestion
    arr = pd.array(["a", "b"], dtype="string")
```

is a bit shorter to construct those